docs/user-guide/installation.md

```diff
@@ -96,7 +96,7 @@ pip install 'polars[numpy,fsspec]'
 ```toml
 # Cargo.toml
 [dependencies]
-polars = { version = "0.26.1", features = ["lazy", "temporal", "describe", "json", "parquet", "dtype-datetime"] }
+polars = { version = "0.39", features = ["lazy", "temporal", "describe", "json", "parquet", "dtype-datetime"] }
```

Comment:
Can we make this `"*"`. Otherwise it will be outdated very soon.

Reply:
updated